.github/workflows/runSchematronValidation.yml

```diff
@@ -25,6 +25,15 @@ jobs:
     steps:
       # Check-out the repository under $GITHUB_WORKSPACE
       - uses: actions/checkout@v2
+
+      - name: Cache
+        uses: actions/cache@v2.1.3
+        with:
+          path: |
+            ~/resources/validations/lib/
+            ~/oscal/
+            /tmp/saxon/
+          key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
```

Comment:
Have we checked the applicability of this keying strategy?  We have no files with the name `lockfiles` in any subdirectory, but we are checking for things that are in sub-modules, we might want to combine all the hashes of the respective sub-modules, or hash `.gitmodules` which probably will do the same thing.

Additionally, are we using `/tmp/saxon` anymore?

Reply:
no we haven't this is a WIP but it doesn't hur the merge. I can back it off but started to use the existing branch for the sh improvements. I probably should have just started a new branch and stashed my change